### PR TITLE
Add AAC fmtp

### PR DESCRIPTION
**Pull request by Noarkhh**
I think it's worth considering whether we could implement parsing fmtp params more concisely, than having a separate function call for each one

### CODE REVIEW

File: lib/ex_sdp/attribute/fmtp.ex [hunk 1]
```diff
@@ -336,6 +370,86 @@ defmodule ExSDP.Attribute.FMTP do
...
+    with {:ok, value} <- Utils.parse_numeric_string(value),
+         do: {rest, %{fmtp | streamtype: value}}
+  end
+
+  defp parse_param(["config=" <> value | rest], fmtp) do
+    with {:ok, value} <- Utils.parse_config(value),
+         do: {rest, %{fmtp | config: value}}
+  end
+
+  defp parse_param(["mode=" <> value | rest], fmtp) do
```
Comment (varsill) [inline lib/ex_sdp/attribute/fmtp.ex:383]: Out of curiosity: are those field names guaranteed to be unique? (By which I mean - is it guaranteed, that there is no other RFC, that adds `mode` field for fmtp parameter)

**Review by varsill — APPROVED** [review]
Looks good to me, though I'm not much familiar with RFC 3640 :D 
🥇

**Comment by varsill** [comment]
> I think it's worth considering whether we could implement parsing fmtp params more concisely, than having a separate function call for each one

I think we could use the struct we already have (along with some type specification) as a `scheme` description, and then write a function that based on that scheme would parse the incoming binary. But I guess it is out of the scope of this PR :D

File: lib/ex_sdp/attribute/fmtp.ex (hunk 1 above)
Comment (Noarkhh) [reply lib/ex_sdp/attribute/fmtp.ex:383]: no, even in this PR this is visible. The RFC3640 specifies a `profile-level-id`, but such a field is already defined in H264 section in the struct, so I don't think we need a separate one.

**Pull request merged**